### ISSUE TITLE
Process market data and compute first technical indicator

### ISSUE THREAD

**Issue by amuradon** [amuradon/sigpron#59]
Consume and process market data like ticket, order book etc.
Make sure this goes along with backtesting as there are limitations for historical data. If I would miss some area in historical data then the results would not accurate.

**Comment by amuradon** [comment]
I can get current as well as historic K-line data for last, index and mark price easily throw REST API. Using startTime and endTime the endpoint allows to get some portion of historical data that is even necessary for real-time technical indicator computation. Getting those historical data for backtesting is straigthforward as well.
The functionality would be like this
* Live / Paper trading
  * Signal comes in, the system subsrcibes for K-line update of given symbol as well as requests data to past since now, e.g. like a day back, draws a chart on those with technical indicators
* Backtesting
  * It would behave the same as above but the source of data would come from REST API endpoint simulator

**Comment by amuradon** [comment]
I was actually thinking if I could potentially use purely client-side charting library with technical indicators to calculate them for me. But there are 2 problems with that idea
* There would need to be browser running as it's client-side library
* How to get technical indicators data out of the library (not sure if any of them has such capability)

For the first one there might be the solution to run it in browser headless mode or using HtmlUnit or such things. Not sure how reliable that would be.
Basically I need all the calculation happen in back-end to process, compute and react even without any chart displayed. The chart is for my convenience to see what happened.
